### PR TITLE
PAS-20 Add nav bar

### PR DESCRIPTION
**Pull request by pawelkondraciuk**

### CODE REVIEW

**Comment by pawelkondraciuk** [comment]
[Add header](https://trello.com/c/rZUvbZ8m/20-add-header)

File: apps/frontend/src/app/NavBar/NavBar.tsx [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+import { matchPath } from 'react-router';
+import { NavLink } from 'react-router-dom';
+
+const navigation = [
+  { name: 'Dashboard', href: '' },
+  { name: 'Balances', href: 'balances' },
+  { name: 'Transfers', href: 'transfers' },
+];
+
+function classNames(...classes: string[]) {
```
Comment (lukszymanski) [inline apps/frontend/src/app/NavBar/NavBar.tsx:14]: we could use one of two
1. https://www.npmjs.com/package/classnames 
2. https://github.com/lukeed/clsx
both good

File: apps/frontend/src/app/NavBar/NavBar.tsx [hunk 2]
```diff
@@ -0,0 +1,161 @@
...
+  return classes.filter(Boolean).join(' ');
```
Comment (lukszymanski) [inline apps/frontend/src/app/NavBar/NavBar.tsx:15]: for class manipulation we could use  https://github.com/lukeed/clsx or https://www.npmjs.com/package/classnames

File: apps/frontend/src/app/NavBar/NavBar.tsx [hunk 3]
```diff
@@ -0,0 +1,161 @@
...
+}
+
+export function NavBar() {
+  const auth = useAuth();
+  const logoutHandler = useCallback(() => {
+    auth.signOut();
+  }, [auth]);
+  const isCurrent = (href: string) => {
```
Comment (lukszymanski) [inline apps/frontend/src/app/NavBar/NavBar.tsx:23]: add useMemo here

Comment (pawelkondraciuk) [reply apps/frontend/src/app/NavBar/NavBar.tsx]: Are you sure? See usage in the template, I've used useCallback instead

**Review by lukszymanski — APPROVED** [review]

**Pull request merged**